cpp/README

```diff
@@ -1,16 +1,17 @@
 = Keyczar C++ =
 
-== Dependancies ==
+== Dependencies ==
 
 - C++ compiler
 - Python for running the build system
 - OpenSSL (>= 0.9.8b)
+- gtest (>= 1.4)
 - [Optional]: Zlib (for compression)
 - [Optional]: Swig (Python binding)
 
 === Ubuntu/Debian ===
 
-Packages g++ make libssl-dev zlib1g-dev python python-dev swig
+Packages g++ make libssl-dev zlib1g-dev libgtest-dev python python-dev swig
 
 === MacOSX Snow Leopard ===
 
```

Comment:
Does the macosx package list need to change too?

I can test this out on my other computer if you don't have one available.